include/zephyr/bluetooth/audio/bap.h

```diff
@@ -35,11 +35,252 @@
+	defined(__DOXYGEN__)
+		/**
+		 * @brief Maximum Transport Latency
+		 *
+		 * Not used for the @kconfig{CONFIG_BT_BAP_BROADCAST_SINK} role.
+		 */
+		uint16_t latency;
+#endif /*  CONFIG_BT_BAP_BROADCAST_SOURCE || CONFIG_BT_BAP_UNICAST */
+
+		/**
+		 * @brief SDU Interval
+		 *
+		 * Value range @ref BT_ISO_SDU_INTERVAL_MIN to @ref BT_ISO_SDU_INTERVAL_MAX
+		 */
+		uint32_t interval;
+
+#if defined(CONFIG_BT_ISO_TEST_PARAMS) || defined(__DOXYGEN__)
+		/**
+		 * @brief Maximum PDU size
+		 *
+		 * Maximum size, in octets, of the payload from link layer to link layer.
+		 *
+		 *  Value range @ref BT_ISO_CONNECTED_PDU_MIN to @ref BT_ISO_PDU_MAX for
+		 *  connected ISO.
+		 *
+		 *  Value range @ref BT_ISO_BROADCAST_PDU_MIN to @ref BT_ISO_PDU_MAX for
+		 *  broadcast ISO.
+		 */
+		uint16_t max_pdu;
+
+		/**
+		 * @brief Burst number
+		 *
+		 * Value range @ref BT_ISO_BN_MIN to @ref BT_ISO_BN_MAX.
+		 */
+		uint8_t burst_number;
+
+		/**
+		 * @brief Number of subevents
+		 *
+		 * Maximum number of subevents in each CIS or BIS event.
+		 *
+		 * Value range @ref BT_ISO_NSE_MIN to @ref BT_ISO_NSE_MAX.
+		 */
+		uint8_t num_subevents;
+#endif /* CONFIG_BT_ISO_TEST_PARAMS */
+	};
+};
+
+/**
+ * @brief Helper to declare elements of @ref bt_bap_qos_cfg_pref
+ *
+ * @param _unframed_supported Unframed PDUs supported
+ * @param _phy Preferred Target PHY
+ * @param _rtn Preferred Retransmission number
+ * @param _latency Preferred Maximum Transport Latency (msec)
+ * @param _pd_min Minimum Presentation Delay (usec)
+ * @param _pd_max Maximum Presentation Delay (usec)
+ * @param _pref_pd_min Preferred Minimum Presentation Delay (usec)
+ * @param _pref_pd_max Preferred Maximum Presentation Delay (usec)
+ */
+#define BT_BAP_QOS_CFG_PREF(_unframed_supported, _phy, _rtn, _latency, _pd_min, _pd_max,           \
+			    _pref_pd_min, _pref_pd_max)                                            \
+	{                                                                                          \
+		.unframed_supported = _unframed_supported, .phy = _phy, .rtn = _rtn,               \
+		.latency = _latency, .pd_min = _pd_min, .pd_max = _pd_max,                         \
+		.pref_pd_min = _pref_pd_min, .pref_pd_max = _pref_pd_max,                          \
+	}
+
+/** @brief Audio Stream Quality of Service Preference structure. */
+struct bt_bap_qos_cfg_pref {
+	/**
+	 * @brief Unframed PDUs supported
+	 *
+	 *  Unlike the other fields, this is not a preference but whether
+	 *  the codec supports unframed ISOAL PDUs.
+	 */
+	bool unframed_supported;
+
+	/** Preferred PHY */
+	uint8_t phy;
+
+	/** Preferred Retransmission Number */
+	uint8_t rtn;
+
+	/** Preferred Transport Latency */
+	uint16_t latency;
+
+	/**
+	 * @brief Minimum Presentation Delay in microseconds
+	 *
+	 * Unlike the other fields, this is not a preference but a minimum requirement.
+	 *
+	 * Value range 0 to @ref BT_AUDIO_PD_MAX, or @ref BT_AUDIO_PD_PREF_NONE
+	 * to indicate no preference.
```

Comment:
The documentation for the presentation delay fields seem wrong. Although fixing this is not strictly part of this PR it may as well be done here, instead of creating a separate PR for fixing it.

```suggestion
	 * Unlike the other fields, this is not a preference but a minimum requirement.
	 *
	 * Value range 0 to @ref BT_AUDIO_PD_MIN
```